src/java/main/org/apache/zookeeper/server/quorum/QuorumPeerConfig.java

```diff
@@ -750,4 +761,46 @@ public static void setReconfigEnabled(boolean enabled) {
         reconfigEnabled = enabled;
     }
 
+    /**
+     * @return the {@code RateLimiter} to use
+     */
+    public static String getRateLimiterImpl() {
```

Comment:
I am not so sure that creating a couple of static fields is the best approach here...